enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -110,12 +131,12 @@
   (when-not (empty? escaped-dashcards)
     (println "Dashboard cards outside the collection")
     (println "======================================")
-    (doseq [[dash-id dashcards] escaped-dashcards
+    (doseq [[dash-id card-ids] escaped-dashcards
             :let [dash-name (db/select-one-field :name Dashboard :id dash-id)]]
```

Comment:
I guess since this was already in here you don't need to change it in this PR but this is another case where you could fetch everything in one query using `:id [:in ...]]` and **then** iterate over the rows to avoid an `N+1` query situation. It looks like we have a lot of N+1s going on so fixing them all would probably improve performance pretty significantly.

In some places you might be able to use `db/select-reducible` instead of `db/select` and then reduce the results which would be even more performant, since it would handle the rows (e.g. print them or whatever you wanted to do) as they come in instead of having to get everything in memory all at once